src/main/java/cubicchunks/worldgen/GeneratorPipeline.java

```diff
@@ -102,10 +102,6 @@ public void generate(Cube cube, GeneratorStage targetStage) {
 			if (dependency != null) {
 				Dependent dependent = new Dependent(cube, dependency);
 				this.dependencyManager.register(dependent);
-
-				if (dependent.isSatisfied()) {
-					stage.getProcessor().processor.add(cube.getAddress());
-				}
```

Comment:
This is not required due to DependencyManager.register(...) queuing the cube if all dependencies are satisfied.